src/main/java/frc/robot/component/Climber.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.component;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.units.Current;
+import edu.wpi.first.units.Time;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInLayouts;
+import frc.robot.util.MoPrefs;
+import frc.robot.util.MoPrefs.UnitPref;
+import frc.robot.util.MoShuffleboard;
+import frc.robot.util.MoUnits;
+
+/* A single side climber */
+public class Climber {
+    public final CANSparkMax winch;
+    public final SparkPIDController winchPID;
+    public final CurrentTrigger zeroLimit;
+    private boolean hasZero = false;
+
+    public Climber(
+            String name, CANSparkMax winch, UnitPref<Current> zeroCurrentLimit, UnitPref<Time> zeroTriggerDuration) {
+        this.winch = winch;
+        this.winchPID = winch.getPIDController();
```

Comment:
No, we don't need pid on this. Straight motor power should be perfectly fine, and we don't have time to waste tuning unnecessary pid loops.